app/models/response/ResponseArrival.scala

```diff
@@ -18,15 +18,16 @@ package models.response
 
 import java.time.LocalDateTime
 
-import models.domain.{Arrival}
+import models.domain.Arrival
 import play.api.libs.json.Json
+import utils.Utils
 
 object ResponseArrival {
 
   implicit val format = Json.format[ResponseArrival]
 
   def apply(a: Arrival): ResponseArrival = {
```

Comment:
Can this method take in an extra argument for location and then you could pass in something like routes.ArrivalMovementController.getArrivals(arrivalId).url ? It's just to avoid hardcoding the path